javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -81,30 +81,10 @@ public static void installBytebuddyAgent(Instrumentation inst) {
   public static ResettableClassFileTransformer installBytebuddyAgent(
       Instrumentation inst,
       boolean skipAdditionalLibraryMatcher,
+      Iterable<ComponentInstaller> componentInstallers,
       AgentBuilder.Listener... listeners) {
 
-    ClassLoader savedContextClassLoader = Thread.currentThread().getContextClassLoader();
-    try {
-      // calling (shaded) OpenTelemetry.getGlobalTracerProvider() with context class loader set to
-      // the
-      // agent class loader, so that SPI finds the agent's (isolated) SDK, and (shaded)
-      // OpenTelemetry registers it, and then when instrumentation calls (shaded)
-      // OpenTelemetry.getGlobalTracerProvider() later, they get back the agent's (isolated) SDK
-      //
-      // but if we don't trigger this early registration, then if instrumentation is the first to
-      // call (shaded) OpenTelemetry.getGlobalTracerProvider(), then SPI can't see the agent class
-      // loader,
-      // and so (shaded) OpenTelemetry registers the no-op TracerFactory, and it cannot be replaced
-      // later
```

Comment:
Can you preserve this comment in `OpenTelemetryInstaller`?

Reply:
moved